channels/applicationinsights-channel-js/api-extractor.json

```diff
@@ -0,0 +1,361 @@
+/**
+ * Config file for API Extractor.  For more info, please visit: https://api-extractor.com
+ */
+ {
+    "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
```

Comment:
Adding more Api Extractor generations -- VERY useful for spotting the missed exports